macos/package_files/3.9.3/postinstall.sh

```diff
@@ -14,6 +14,12 @@ DIR="/Library/Ossec"
 INSTALLATION_SCRIPTS_DIR="${DIR}/packages_files/agent_installation_scripts"
 SCA_FILES_DIR="${INSTALLATION_SCRIPTS_DIR}/sca"
 
+if [ -d $DIR/config_files ]; then
+    rm -rf $DIR/etc/{ossec.conf,client.keys,local_internal_options.conf,shared}
```

Comment:
Change `$DIR` to `${DIR}` please. It is easier to see that DIR is actually a variable.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/12a6f23b764a12427b2722ee406a485c9d05576f

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/835644bb4e4316e8a8b751f9dd9192ea305f6352  and https://github.com/wazuh/wazuh-packages/pull/221/commits/12a6f23b764a12427b2722ee406a485c9d05576f